src/components/Button.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Button.css';
+
+const Button = (props) => {
+  const { buttonName } = props;
+  const { color } = props;
+  const { wide } = props;
+  return wide ? (
+    <div className="button-component" style={{ width: '50%' }}>
+      <button type="button" style={{ backgroundColor: color }}>
+        {buttonName}
+      </button>
+    </div>
+  ) : (
+    <div className="button-component" style={{ width: '25%' }}>
+      <button type="button" style={{ backgroundColor: color }}>
+        {buttonName}
+      </button>
+    </div>
+  );
+};
```

Comment:
[Optional]

We keep the code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) by doing this. :thinking: 

```diff
-  return wide ? (
+ (
-    <div className="button-component" style={{ width: '50%' }}>
+    <div className="button-component" style={{ width: wide? '50%' : '25%' }}>
      <button type="button" style={{ backgroundColor: color }}>
        {buttonName}
      </button>
    </div>
-  ) : (
-    <div className="button-component" style={{ width: '25%' }}>
-      <button type="button" style={{ backgroundColor: color }}>
-        {buttonName}
-      </button>
-    </div>
  );
```